cmd/version.go

```diff
@@ -0,0 +1,30 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	"github.com/spf13/cobra"
+)
+
+const SupportedConfigVersion = "v1"
+
+func init() {
+	rootCmd.AddCommand(versionCmd)
+}
+
+var versionCmd = &cobra.Command{
+	Use:     "version",
+	Short:   "Print the reposeed and config versions",
+	Example: "reposeed version",
+	Run:     versionHandler,
+}
+
+func versionHandler(cmd *cobra.Command, args []string) {
+	reposeedVersion, err := ioutil.ReadFile("VERSION")
```

Comment:
That won't work as the VERSION file is not always in the same directory.
This seems to be the initial version of the version command, which evolved quite a bit see the PR: https://github.com/resumic/schema/pull/95